index.html

```diff
@@ -9399,7 +9504,7 @@ <h2>Translatable States and Properties</h2>
 	</section>
 	<section id="global_states">
 		<h2>Global States and Properties</h2>
-		<p>Some <a>states</a> and <a>properties</a> are applicable to all host language <a>elements</a> regardless of whether a <a>role</a> is applied. The following global states and properties are supported by all roles and by all base markup elements.</p>
+		<p>Some <a>states</a> and <a>properties</a> are applicable to all host language <a>elements</a> regardless of whether a <a>role</a> is applied. The following global states and properties are supported by all roles and by all base markup elements unless otherwise prohibited. Any global states and properties which are prohibited on any role will be listed in the Prohibited States and Properties section.</p>
 		<p class="placeholder">Placeholder for global states and properties</p>
 		<p>Global states and properties are applied to the role <rref>roletype</rref>, which is the base role, and therefore inherit into all roles. To facilitate reading, they are not explicitly identified as either supported or inherited states and properties in the specification. Instead, the inheritance is indicated by a link to this section.</p>
```

Comment:
I think this paragraph should be deleted. It is confusing. The reality is that global states and properties are included in the list of inherrited states and properties in every role definition.

In addition, line 10937, in the definition of aria-label had the following in the row for "used in roles":

><td class="property-applicability">All elements of the base markup</td>

Should be changed to:

<td class="property-applicability">All elements of the base markup except for some elements that prohibit its use</td>

The same needs to be done for aria-labelledby.